tests/gpts/github/test-config/config.ts

```diff
@@ -1,76 +1,74 @@
 import { z } from "zod";
+import * as fs from 'fs';
+import * as path from 'path';
 
 interface softwareTemplatesConfig {
     templates: string[];
+    priority: string[];
     github: {
-        active: boolean;
         tekton: boolean;
         jenkins: boolean;
         actions: boolean;
         host: string;
-        registriesConfig: {
-            quay: {
-                active: boolean;
-                host: string;
-            };
-        };
     };
     gitlab: {
-        active: boolean;
         tekton: boolean;
         jenkins: boolean;
         gitlabci: boolean;
         host: string;
-        registriesConfig: {
```

Comment:
how are we fixing the registries?

Reply:
I don't think we need it now

Reply:
ok so you are handling it via pict file?

Reply:
no, for the current automation framework, it doesn't need to handle it. Regarding registry, it gets it from environment variables. each time, it only can handle one of registries.